<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockGlyphEdit</class>
 <widget class="QDockWidget" name="DockGlyphEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glyph Edit &amp; Preview</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBoxGridParams">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Add Grid Cells</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxLeftCells">
         <property name="title">
          <string>Left </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSpinBox" name="spinBoxLeftCells"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTopCells">
         <property name="title">
          <string>Top</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSpinBox" name="spinBoxTopCells"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxRightCells">
         <property name="title">
          <string>Right</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QSpinBox" name="spinBoxRightCells"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxBottomCells">
         <property name="title">
          <string>Bottom</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSpinBox" name="spinBoxBottomCells"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxBaseLines">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Baselines</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonBaselineMoveTop">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="glyph.qrc">
           <normaloff>:/baseline/up</normaloff>:/baseline/up</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonBaselineMoveLeft">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="glyph.qrc">
           <normaloff>:/baseline/left</normaloff>:/baseline/left</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonBaselinesReset">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="glyph.qrc">
           <normaloff>:/baseline/reset</normaloff>:/baseline/reset</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButtonBaselineRight">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="glyph.qrc">
           <normaloff>:/baseline/right</normaloff>:/baseline/right</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButtonBaselineBottom">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="glyph.qrc">
           <normaloff>:/baseline/down</normaloff>:/baseline/down</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBoxGlyphPreview">
       <property name="title">
        <string>Glyph Preview</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QWidget" name="widgetGlyphPreview" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxBitmapDimension">
       <property name="title">
        <string>Bitmap Dimension</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QListView" name="listViewBitmapDimensions"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxStringPreview">
       <property name="title">
        <string>String Preview</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="lineEditStringPreview"/>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableGlyphs"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="glyph.qrc"/>
 </resources>
 <connections/>
</ui>
